wagtail/images/tests/test_models.py

```diff
@@ -346,6 +352,32 @@ def test_background_position_style_default(self):
             rendition.background_position_style, "background-position: 50% 50%;"
         )
 
+    def test_custom_rendition_backend(self):
+        """
+        Unfortunately, the storage setting is loaded at startup.
+        Overriding it is cumbersome, let's just make sure changes are contained.
+        """
+        # when setting is not set, instance.get_storage() returns DefaultStorage
+        rendition1 = self.image.get_rendition("min-120x120")
+        self.assertIsInstance(get_rendition_storage(rendition1), DefaultStorage)
+        self.assertIsInstance(rendition1.get_storage(), DefaultStorage)
+
+        # when setting is set, instance.get_storage() returns the custom storage backend
+        from django.conf import settings
+
+        bkp = settings
+        setattr(settings, "WAGTAILIMAGES_RENDITION_STORAGE", CustomStorage())
```

Comment:
If you use the override settings similar to other parts of the code - `@override_settings(WAGTAIL_USAGE_COUNT_ENABLED=True)`

this may work easier and avoid the loading at start up issue

Reply:
Using the decorator won't work in this case.  Since this gets defined for the class when models are loaded, that setting has to be defined at startup, or it has no effect.  It can't be overridden at runtime, we'd have to call django.setup() again.  TestCase.setUp() runs after models are loaded, so that's also not a possibility. I have updated the comments on the test case to be more specific as to the problem.